README.md

```diff
@@ -14,7 +14,7 @@ so using the Crossplane CLI in a Kubernetes cluster where Crossplane is
 installed:
 
 ```console
-kubectl crossplane install provider crossplanecontrib/provider-kubernetes:main
+kubectl crossplane xpkg install provider xpkg.upbound.io/crossplane-contrib/provider-kubernetes:v0.13.0
```

Comment:
The [Crossplane CLI](https://docs.crossplane.io/latest/cli/) is no longer a plugin for `kubectl`, so we can remove the `kubectl` from this line as well. 

```suggestion
crossplane xpkg install provider xpkg.upbound.io/crossplane-contrib/provider-kubernetes:v0.13.0
```